go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -622,16 +623,16 @@ func setupCustomerKeyspace(t *testing.T) {
 	require.NoError(t, err)
 	err = cluster.WaitForHealthyShard(vc.VtctldClient, "customer", "80-")
 	require.NoError(t, err)
-	if err := vtgate.WaitForStatusOfTabletInShard(fmt.Sprintf("%s.%s.replica", "customer", "-80"), 2); err != nil {
+	if err := vtgate.WaitForStatusOfTabletInShard(fmt.Sprintf("%s.%s.replica", "customer", "-80"), 2, 30*time2.Second); err != nil {
```

Comment:
nitpick: while you're here, you might as well have switched to `require.NoError`

Reply:
Fixed via 013d4047e5